### PR TITLE
minor constant cleanup

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
a little constant and comment cleanup
#### Summary of Changes

Fixes #

### CODE REVIEW

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -5234,21 +5238,15 @@ impl AccountsDb {
...
     ) -> Result<(Hash, u64), BankHashVerificationError> {
         let mut scan_and_hash = move || {
-            // # of passes should be a function of the total # of accounts that are active.
-            // higher passes = slower total time, lower dynamic memory usage
-            // lower passes = faster total time, higher dynamic memory usage
-            // passes=2 cuts dynamic memory usage in approximately half.
-            let num_scan_passes: usize = 2;
-
-            let bins_per_pass = PUBKEY_BINS_FOR_CALCULATING_HASHES / num_scan_passes;
+            let bins_per_pass = PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES;
```
Comment (brooksprumo) [inline runtime/src/accounts_db.rs:5241]: Why not just use `BINS_PER_PASS` here?

File: runtime/src/accounts_db.rs [hunk 2]
```diff
@@ -88,6 +88,10 @@ pub const DEFAULT_NUM_DIRS: u32 = 4;
 // When calculating hashes, it is helpful to break the pubkeys found into bins based on the pubkey value.
 // More bins means smaller vectors to sort, copy, etc.
 pub const PUBKEY_BINS_FOR_CALCULATING_HASHES: usize = 65536;
+// # of passes should be a function of the total # of accounts that are active.
+// higher passes = slower total time, lower dynamic memory usage
+// lower passes = faster total time, higher dynamic memory usage
+// passes=2 cuts dynamic memory usage in approximately half.
 pub const NUM_SCAN_PASSES: usize = 2;
 pub const BINS_PER_PASS: usize = 32768; // PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES
```
Comment (brooksprumo) [inline runtime/src/accounts_db.rs:96]: And this constant could be calculated too, right?
```suggestion
pub const BINS_PER_PASS: usize = PUBKEY_BINS_FOR_CALCULATING_HASHES / NUM_SCAN_PASSES;
```

File: runtime/src/accounts_db.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/accounts_db.rs]: sure 'nuff. this used to fail in rust const eval, I thought.

**Review by brooksprumo — APPROVED** [review]
lgtm

**Pull request merged**